src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/BinaryFormatUtilitiesTests.cs

```diff
@@ -25,33 +26,58 @@ private void WriteObjectToStream(object value, bool restrictSerialization = fals
     private object? ReadObjectFromStream()
     {
         _stream.Position = 0;
-        return Utilities.ReadObjectFromStream<object>(_stream, resolver: null, legacyMode: true);
+        DataRequest request = new("test")
+        {
+            UntypedRequest = true
+        };
+
+        return Utilities.ReadObjectFromStream<object>(_stream, in request);
     }
 
     private object? ReadRestrictedObjectFromStream()
     {
         _stream.Position = 0;
-        return Utilities.ReadRestrictedObjectFromStream<object>(_stream, resolver: null, legacyMode: true);
+        DataRequest request = new("test")
+        {
+            UntypedRequest = true
+        };
+
+        return Utilities.ReadRestrictedObjectFromStream<object>(_stream, in request);
     }
 
     private object? ReadObjectFromStream<T>(bool restrictDeserialization, Func<TypeName, Type>? resolver)
     {
         _stream.Position = 0;
+        DataRequest request = new("test")
```

Comment:
```suggestion
        DataRequest request = new(restrictDeserialization ? "TEXT" : "test")
```

or DataRequest request = new("unused")

Reply:
As above, "test" is pretty standard.

Reply:
Sure, but elsewhere it's a custom format. This test file is format-agnostic, "unused" is a better choice.